swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -1684,7 +1691,7 @@ paths:
         - $ref: "#/components/parameters/dateFrom"
         - $ref: "#/components/parameters/dateTo"
         - $ref: "#/components/parameters/entryReferenceFrom"
-        - $ref: "#/components/parameters/bookingStatus"
+        - $ref: "#/components/parameters/bookingStatusGeneric"
```

Comment:
Relevant Change: bookingStatusGeneric renamed bookingStatus
Adoption approach: Follow the change